### DISCUSSION TITLE
How to programmatically merge cells

### DISCUSSION THREAD

**Discussion by x68507** [myliang/x-spreadsheet#626] [General]
This is a super simple question, but how can I programmatically merge a range of cells _after_ rendering the data.  I don't want to rely on the user to select the cells and I don't know I want to merge the cells until I get additional data back from the server, so this needs to be done after the **loadData()** method.  

Also, is there an actual published API or tutorial somewhere?  The ReadMe isn't enough to actually do anything useful.

**Comment by reviewher** [comment]
Use getData and refresh the data:

```js
var data = xs.getData();
data[0].merges.push("A1:B2");
xs.loadData(data);
```
